### PR TITLE
Katie homepage qa

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

**Comment by asasmith** [comment]
Container alignment and spacing between the cards seems to be off here. The cards starting position should align with the edge of the "container" element on the left side of the screen. This looks like they start center aligned.

html
screenshot

designs
image
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 91.0.4472.77 on Mac OS 10.15.7
Viewport:  1677 x 914 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-30--airtec.netlify.app%2F&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=1677x914&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-30--airtec.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60c14460520c6c0007401763&commentID=60c146a1e1f3121cdeef56b8&resolution=resolved)

**Comment by asasmith** [comment]
screenshot

The accent is not center aligned. I think you want to update to be 
```
left: 50%;
transform: translate(-50%);
``` 


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 91.0.4472.77 on Mac OS 10.15.7
Viewport:  1677 x 914 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-30--airtec.netlify.app%2F&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=1677x914&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-30--airtec.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60c14460520c6c0007401763&commentID=60c1477bc498c221052ed087&resolution=resolved)

File: web/tailwind.config.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ module.exports = {
     darkMode: false, // or 'media' or 'class'
     theme: {
         extend: {
+            borderRadius: {
+                '4xl': '2.5rem',
+            },
```
Comment (asasmith) [inline web/tailwind.config.js:8]: Do we need this?

**Comment by asasmith** [comment]
Seems like left padding is off here.
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 91.0.4472.77 on Mac OS 10.15.7
Viewport:  1680 x 914 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-30--airtec.netlify.app%2F&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=1680x914&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-30--airtec.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60c17c48dcae320008e753f5&commentID=60c17dcbc498c27d312eccce&resolution=resolved)

File: web/tailwind.config.js (hunk 1 above)
Comment (kcbreuil) [reply web/tailwind.config.js:8]: @asasmith  yes, josh requested we change the border radius to 2.5. TW's class only goes up to 1.5

Comment (asasmith) [reply web/tailwind.config.js:8]: @kcbreuil can't you do the same as you did for the `cardStyle` declaration in `/components/card.js`?

Comment (kcbreuil) [reply web/tailwind.config.js:8]: @asasmith it didnt extend

Comment (asasmith) [reply web/tailwind.config.js:8]: that seems odd. Are you seeing the class applied correctly?

Comment (asasmith) [reply web/tailwind.config.js:8]: `rounded-bl[2.5rem]` seems like it should work in both places.

**Comment by kcbreuil** [comment]
> Seems like left padding is off here.
> screenshot
> 
> Browser metadata
> [Open Deploy Preview](https://deploy-preview-30--airtec.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60c17c48dcae320008e753f5&commentID=60c17dcbc498c27d312eccce&resolution=resolved)

@asasmith not sure why the deploy preview showed that, but it's been resolved

File: web/tailwind.config.js (hunk 1 above)
Comment (kcbreuil) [reply web/tailwind.config.js:8]: Nope, it doesn't. It applies it in the html but doesnt extend. Creating the custom class in the config does. @asasmith

Comment (asasmith) [reply web/tailwind.config.js:8]: it works
Screen Shot 2021-06-10 at 12 20 35 PM

Comment (asasmith) [reply web/tailwind.config.js:8]: Screen Shot 2021-06-10 at 12 21 57 PM

File: web/src/components/card.js [hunk 2]
```diff
@@ -21,14 +21,14 @@ export default function Card({
         : 'bg-white text-brand-1';
     const transparentStyle = isTransparent ? 'bg-transparent text-black' : '';
 
-    const cardStyle = `rounded-bl-lg w-11/12 md:w-7/12 pl-6
+    const cardStyle = `pl-6 rounded-bl-4xl w-11/12 md:w-6/12 
    ${darkStyle} ${transparentStyle}
     ${isRight ? 'md:transform md:translate-x-3/4' : ''}`;
```
Comment (asasmith) [inline web/src/components/card.js:26]: I didn't catch this before but I don't think this is how we want to handle this. Using a combination of `margin-right` and `margin-left` gets the correct alignment.

`${isRight ? 'md:ml-auto' : 'md:mr-auto'}`;

**Review by asasmith — CHANGES_REQUESTED** [review]

File: web/src/components/card.js (hunk 2 above)
Comment (kcbreuil) [reply web/src/components/card.js:26]: @asasmith thats not it.

Comment (asasmith) [reply web/src/components/card.js:26]: @kcbreuil what's not it?

Comment (kcbreuil) [reply web/src/components/card.js:26]: ml-auto does not exist

Comment (asasmith) [reply web/src/components/card.js:26]: Screen Shot 2021-06-10 at 1 53 34 PM

Comment (kcbreuil) [reply web/src/components/card.js:26]: @asasmith Thats not the logic we need--i am not positioning this with margin?

Comment (asasmith) [reply web/src/components/card.js]: you can see that in the image above

**Review by asasmith — CHANGES_REQUESTED** [review]
@kcbreuil please see this [comment](https://docs.google.com/document/d/1O8_wp1rHx2NYPKzXoyCTsTk-sfYuaf-fnYNqH-_C8lg/edit?disco=AAAAMzlyDK0) from Josh about the news list on small screens

**Pull request merged**